packages/frontend/src/pages/FeedbackPage.tsx

```diff
@@ -12,9 +14,12 @@ export default function FeedbackPage() {
   useEffect(() => {
     if (!section || !sk) return;
 
-    const url = `${import.meta.env.VITE_API_URL}/scores/${section}/${sk}`;
-    fetch(url)
-      .then(response => response.json())
+    toJSON(
+      post({
```

Comment:
I think the needed method is get instead of post

Reply:
My bad :)